crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po

```diff
@@ -1183,8 +1183,8 @@ msgid "Any supporting materials (please list and describe them)."
 msgstr "Cualquier material de apoyo (haga una lista y describa cada artículo)."
 
 #: templates/base.html:22 templates/base.html:40
-msgid "Civil Rights Division - Department of Justice"
-msgstr "La División de Derechos Civiles - Departamento de Justicia"
+msgid "Contact the Civil Rights Division | Department of Justice"
+msgstr "Comuníquese con la División de Derechos Civiles - Departamento de Justicia"
```

Comment:
We should use `|` consistently 
```suggestion
msgstr "Comuníquese con la División de Derechos Civiles | Departamento de Justicia"
```

Reply:
Ha. yes, i just copied from the translation.

Reply:
Done, also at report_base.html